src/test/java/com/amazon/opendistroforelasticsearch/sql/util/HasFieldWithValue.java

```diff
@@ -43,7 +42,6 @@ private HasFieldWithValue(String name, String desc, Matcher<? super U> matcher)
         this.fieldName = name;
     }
 
-    @Factory
```

Comment:
Sorry, not very familiar with this annotation. What is the reason we are removing this?

Reply:
Elastic has migrated to new version of harmcrest and this annotation was deprecated.
Seems like this is some old annotation that was used in previous versions of harmcrest. 

As per harmcrest doc:

https://github.com/hamcrest/JavaHamcrest/blob/master/CHANGES.md
org.hamcrest.Factory has been removed. This was only used in old implementations of the hamcrest build toolchain, and can safely be deleted from client code without any effect.